lib/adhoq/storage/cache.rb

```diff
@@ -22,10 +22,6 @@ def direct_download?
       def get(id)
         cache.read(cache_prefix + id)
       end
-
-      def get_url(_report)
-        raise NotImplementedError
-      end
```

Comment:
Why did you remove it?

Reply:
Moved to base class - `lib/adhoq/storage/fog_storage.rb `

Reply:
https://github.com/romikoops/adhoq/blob/master/lib/adhoq/storage/cache.rb#L3 Base class is Object

Reply:
fixed by "revert back get_url to Storage::Cache and Storage::OnTheFly"